### ISSUE TITLE
ERROR: pull access denied for plebnet-playground-docker_dashboard

### ISSUE THREAD

**Issue by thejerrod** [PLEBNET-PLAYGROUND/plebnet-playground-docker#82]
rpi4, raspberrypi os
Linux cluster1 5.10.92-v8+ #1514 SMP PREEMPT Mon Jan 17 17:39:38 GMT 2022 aarch64 GNU/Linux

After  installing via:
TRIPLET=aarch64-linux-gnu ./install.sh

I tried running docker-compose up and received an access denied error:

$ docker-compose up
Pulling notebook (plebnet-playground-docker_dashboard:)...
ERROR: The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.

Continue with the new image? [yN]y
Pulling notebook (plebnet-playground-docker_dashboard:)...
ERROR: pull access denied for plebnet-playground-docker_dashboard, repository does not exist or may require 'docker login': denied: requested access to the resource is denied

I also tried docker compose up:
 $ docker compose up
[+] Running 0/15
 ⠴ lndg Pulling                                                                                                                        3.6s
   ⠴ 39ab78bc09e7 Pulling fs layer                                                                                                     0.6s
   ⠴ 0cf88d09fc28 Pulling fs layer                                                                                                     0.6s
   ⠴ e019dd0368ba Pulling fs layer                                                                                                     0.6s
   ⠴ 4b9f1769d50d Waiting                                                                                                              0.6s
   ⠴ a3cb4eca0ceb Waiting                                                                                                              0.6s
   ⠴ ec372ec4cef1 Waiting                                                                                                              0.6s
   ⠴ f67535cd60ac Waiting                                                                                                              0.6s
   ⠴ e06b94a17a1d Waiting                                                                                                              0.6s
   ⠴ 25160b9ee6e7 Waiting                                                                                                              0.6s
   ⠴ bf5ba8ac2029 Waiting                                                                                                              0.6s
   ⠴ cebf40279c36 Waiting                                                                                                              0.6s
   ⠴ ad5c28565144 Waiting                                                                                                              0.6s
 ⠿ notebook Error                                                                                                                      3.6s
 ⠿ dashboard Error                                                                                                                     3.5s
Error response from daemon: pull access denied for plebnet-playground-docker_dashboard, repository does not exist or may require 'docker login': denied: requested access to the resource is denied

Can you help me troubleshoot?

**Comment by thejerrod** [comment]
If I remove dashboard and notebook the compose file runs

**Comment by rsafier** [comment]
yeah looks like it needs reference put in for its location, not sure how tests missed that, I will fix it

**Comment by rsafier** [comment]
Fixed template with commit ce34b6903a7796b185500f5f00702c6b9d7cc4e7
not sure why but on this mac the install script isn't building properly, but manually running docker compose build is.

**Comment by RandyMcMillan** [comment]
plebnet-playground-docker-dashboard not plebnet-playground-docker_dashboard
